### ISSUE TITLE
Missing root __layout import after upgrade to 2.x

### ISSUE THREAD

**Issue by techniq** [roxiness/routify#260]
After upgrading from `1.9.7` to `2.1.0`, I am getting the following runtime error:

![image](https://user-images.githubusercontent.com/177476/91323677-27113480-e78f-11ea-9b24-2c69b0d7075e.png)

![image](https://user-images.githubusercontent.com/177476/91324185-c7ffef80-e78f-11ea-92f0-ca4d936e01fe.png)

When I look inside the new `.routify/routes.js`, I do not see `__layout` in my imports...

![image](https://user-images.githubusercontent.com/177476/91324593-3fce1a00-e790-11ea-9563-b221836be69e.png)

but if I manually add it, it fixes the error...

![image](https://user-images.githubusercontent.com/177476/91324795-72781280-e790-11ea-8951-e6945947059c.png)

Note: It does appear to pick up a nested `_layout` though...

![image](https://user-images.githubusercontent.com/177476/91326700-a48a7400-e792-11ea-9b5d-f1b903fd1568.png)

This is a single page application (no SSR) served via `sirv`.  To upgrade, I did the following:

- Update version installed in `package.json`
- Replace all imports using `@sveltech/routify` with `@roxi/routify`
- Update `App.svelte` to use generated `../.routify/routes` as seen [here](https://github.com/roxiness/routify-site-2020/blob/master/src/App.svelte#L4)

```diff
- import { Router } from '@sveltech/routify';
- import { routes } from '@sveltech/routify/tmp/routes';
+ import { Router } from '@roxi/routify';
+ import { routes } from '../.routify/routes';
```
- Turned off `dynamicImports` in `package.json` since SPA (and was getting `[!] Error: UMD and IIFE output formats are not supported for code-splitting builds.` error if not)
```
"routify": {
    "dynamicImports": false
  }
```

Here is my full `package.json`

```json
{
  "name": "svelte-app",
  "version": "1.0.0",
  "scripts": {
    "build": "rollup -c",
    "analyze": "source-map-explorer 'public/build/*.js'",
    "dev": "routify -c rollup-watch",
    "rollup-watch": "rollup -c -w",
    "start": "sirv public -s -p 3000",
    "format": "prettier --write \"src/**/*\"",
    "validate": "svelte-check"
  },
  "devDependencies": {
    "@fullhuman/postcss-purgecss": "^2.3.0",
    "@rollup/plugin-commonjs": "^15.0.0",
    "@rollup/plugin-node-resolve": "^9.0.0",
    "@rollup/plugin-replace": "^2.3.3",
    "@rollup/plugin-typescript": "^5.0.2",
    "@roxi/routify": "^2.1.0",
    "@tsconfig/svelte": "^1.0.10",
    "mdsvex": "^0.8.5",
    "postcss": "^7.0.32",
    "postcss-load-config": "^2.1.0",
    "prettier": "^2.0.5",
    "prettier-plugin-svelte": "^1.1.0",
    "rollup": "^2.26.4",
    "rollup-plugin-livereload": "^1.0.0",
    "rollup-plugin-svelte": "^5.0.3",
    "rollup-plugin-terser": "^7.0.0",
    "svelte": "^3.24.1",
    "svelte-check": "^1.0.20",
    "svelte-preprocess": "^4.1.1",
    "tailwind-css-variables": "^2.0.3",
    "tailwindcss": "^1.7.2",
    "tailwindcss-elevation": "^0.3.5",
    "tslib": "^2.0.1",
    "typescript": "^3.9.7"
  },
  "dependencies": {
    "@mdi/js": "^5.5.55",
    "@urql/devtools": "^2.0.2",
    "@urql/svelte": "^0.4.0",
    "d3-format": "^1.4.4",
    "d3-time": "^2.0.0",
    "date-fns": "^2.15.0",
    "graphql": "^15.3.0",
    "graphql-tag": "^2.11.0",
    "lodash": "^4.17.20",
    "oidc-client": "^1.10.1",
    "sirv-cli": "^1.0.6",
    "source-map-explorer": "^2.4.2"
  },
  "routify": {
    "dynamicImports": false
  }
}
```

Thanks for any help.

**Comment by jakobrosenberg** [comment]
I was unable to replicate the issue. Do you have a repo I can see?

**Comment by techniq** [comment]
@jakobrosenberg Let me try to boil it down and I'll set one up.  Probably be a day or two before I can.  Thanks.

**Comment by techniq** [comment]
@jakobrosenberg I think I got this boiled down pretty good (and still getting error):

- https://github.com/techniq/routify-issue-260

**Comment by jakobrosenberg** [comment]
Thanks for reporting this. Should be fixed in `2.1.2`.

**Comment by techniq** [comment]
@jakobrosenberg Confirmed fixed.  Thanks for the quick turnaround.
